clustertemplates/docker-all.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "docker-all",
+  "description": "Docker services on Linux",
+  "defaults": {
+    "services": [
+      "base",
+      "docker"
+    ],
+    "provider": "rackspace",
+    "hardwaretype": "standard-medium",
+    "imagetype": "ubuntu12",
+    "config": {}
+  },
+  "compatibility": {
+    "hardwaretypes": [
+      "standard-2xlarge",
+      "standard-large",
+      "standard-medium",
+      "standard-small",
+      "standard-xlarge"
+    ],
+    "imagetypes": [
+      "centos6",
+      "centos7",
+      "ubuntu12",
+      "ubuntu14"
+    ],
+    "services": [
+      "base",
+      "cassandra-docker",
+      "cdap-standalone",
```

Comment:
Actually, it doesn't say docker.

https://github.com/caskdata/coopr-templates/blob/develop/services/coopr-standalone.json